buildstockbatch/gcp/gcp.py

```diff
@@ -937,40 +939,26 @@ def start_combine_results_job_on_cloud(self, results_dir, do_timeseries=True):
     https://console.cloud.google.com/storage/browser/{self.gcs_bucket}/{self.gcs_prefix}/results/
 
 Run this script with --clean to clean up the GCP environment after post-processing is complete."""
-                )
-                break
-            except:
-                attempts_remaining -= 1
-                if attempts_remaining > 0:
-                    # retry after delay
-                    logger.warning(
-                        "Post-processing Cloud Run job failed to start. "
-                        f"{attempts_remaining} attempt(s) remaining. "
-                        "Will retry in 1 second...",
-                        exc_info=logger.isEnabledFor(logging.DEBUG),
-                    )
-                    time.sleep(1)
-                    continue
-
-                # no attempts remaining
-                logger.warning(
-                    "Post-processing Cloud Run job failed to start after three attempts. "
-                    "You may want to investigate why and try starting it at the console: "
-                    f"{self.postprocessing_job_console_url}",
-                    exc_info=True,
-                )
-                return
+            )
```

Comment:
This makes it much simpler to follow but I feel like we should probably keep the retry and probably change the wait from 1 second to something more robust? Curious to hear your thoughts on removing the retry

Reply:
The retries were only there to handle the case where the job took a few seconds to be created, causing the first attempt(s) to fail. But now the call to `op.result()` above explicitly waits for the job creation to finish. So it's unlikely that just retrying would help anymore.